Changes

```diff
@@ -1,6 +1,59 @@
 OCaml 4.07
 ----------
 
+Highlights of this release include:
```

Comment:
Nit: ~~include~~ (highlights of highlights otherwise?)

Reply:
I don't want to suggest that these are the only entries worth highlighting. I rewrote this to "Some highlights of this release:", but I am not sure that is an improvement over "Highlights ... include:".

Reply:
I would say "Highlights are" or "Novelties include" but not "Highlights include".